manifests/init.pp

```diff
@@ -54,7 +54,6 @@
   $service_template       = $wso2is::params::service_template,
   $ipaddress              = $wso2is::params::ipaddress,
   $enable_secure_vault    = $wso2is::params::enable_secure_vault,
-  $secure_vault_configs   = $wso2is::params::secure_vault_configs,
```

Comment:
I think it is better not to break backward compatibility by removing parameters. If this parameter is removed AFAIU, we might need to remove it from the wso2base/init.pp.